test/compatibility/compatibility_test_for_releases.sh

```diff
@@ -485,6 +485,15 @@ upgrade_downgrade()
                 echo "transaction.timestamps=1" >> $config
                 echo "transaction.implicit=0" >> $config
             fi
+	    # Older releases (8.0 and older) expect to find a BACKUP.copy
+	    # directory if BACKUP exists. After 8.0 the directory structure
+	    # changed. So copy it for older releases if testing against a
+	    # develop run that is doing backups.
+	    dir2=$top/$format_dir_branch2/RUNDIR.$am
+	    if [ "$2" == "develop" && -e $dir2/BACKUP ]; then
```

Comment:
Is there a need to check the non-existence of the `BACKUP.copy` directory before copying? I'm thinking of upgrade/downgrade testing against 2 older releases (e.g. 8.0 and 7.0) where the BACKUP.copy directory has been created.

Reply:
Oops, I missed the 1st check `"$2" == "develop"`. Thinking ahead the future when 9.0 becomes available, do we need to update the 1st check to include 9.0? It feels more portable to check the existence of the "BACKUP.copy" directory than the branch name.

Reply:
Those are good points. I will fix it for moving forward to check the existence of the copy directory.